tests/integration/api/auth_api_test.py

```diff
@@ -86,9 +86,9 @@ def create_test_user(session):
 )
 def test_valid_auth_inputs(login_inputs, expected_result, expected_status_code):
     """ Test authorization API endpoint with valid inputs """
-    with requests.Session() as s:
-        create_test_user(s)
-        response = s.post(
+    with requests.Session() as s_s:
```

Comment:
if it doesn't like 's' please change this to 'session'

Reply:
Updated to session!